config/application.rb

```diff
@@ -156,6 +156,7 @@ class Application < Rails::Application
     config.after_initialize do
       Vmdb::Initializer.init
       ActiveRecord::Base.connection_pool.release_connection
+      puts "ManageIQ version: #{Vmdb::Appliance.VERSION} codename: #{Vmdb::Appliance.CODENAME}"
```

Comment:
`_log.info`?

Reply:
@bdunne 
When I use the log.info it dumps it into the log file but doesn't display it in stdout when you run bin/rails s

INFO -- : MIQ(Vmdb::Application.<class:Application>) ManageIQ version: master codename: H      ammer

I was hoping to see in stdout like puma displays its version string when we run bin/rails s
* Version 3.7.1 (ruby 2.4.4-p296), codename: Snowy Sagebrush

Should I add the log.info as well as puts for the stdout?

Reply:
I have this in my database.yml...
```
puts "** ManageIQ #{evm_version}; Database: adapter=#{c["adapter"]}, name=#{c["database"]}, host=#{c["host"]}"
```
And we also have https://github.com/ManageIQ/manageiq/blob/master/config/initializers/session_store.rb#L43-L46 which gives me the following:
```
11:01:26 bdunne@localhost:~/projects/manageiq (master)$ rails c
** ManageIQ master; Database: adapter=postgresql, name=vmdb_development, host=
** Using session_store: ActionDispatch::Session::MemCacheStore
Loading development environment (Rails 5.0.7)
irb(main):001:0> exit

11:19:10 bdunne@localhost:~/projects/manageiq (master)$ rails s
=> Booting Puma
=> Rails 5.0.7 application starting in development on http://localhost:3000
=> Run `rails server -h` for more startup options
** ManageIQ master; Database: adapter=postgresql, name=vmdb_development, host=
** Using session_store: ActionDispatch::Session::MemCacheStore
I, [2018-07-27T11:19:28.512348 #26973]  INFO -- : Initializing websocket worker!
Puma starting in single mode...
* Version 3.7.1 (ruby 2.4.3-p205), codename: Snowy Sagebrush
* Min threads: 5, max threads: 5
* Environment: development
* Listening on tcp://localhost:3000
Use Ctrl-C to stop
```

Maybe we can combine all of this information somewhere?

Reply:
If you want it in Stdout, user Rails.logger instead.

Reply:
I thinking prefer the following text:

```
puts "** #{Vmdb::Application::PRODUCT_NAME} #{Vmdb::Appliance.VERSION} (Codename {Vmdb::Appliance.CODENAME})"
```

which would look like

```
** ManageIQ master (ManageIQ Hammer)
```

on upstream

Reply:
Additionally, we may want to move this string into a Vmdb::Appliance.BANNER method, then here just do 

```ruby
puts "** #{Vmdb::Appliance.BANNER}"
```